File: Application.Client/src/app/model/education-sync-status.ts
export interface EducationSyncStatus {
  Id?: string;
  LastSyncAt?: number | null;
  ExpiresAt?: number | null;
  Status?: string | null;
  Message?: string | null;
  TriggeredByUserId?: string | null;
  TriggeredByName?: string | null;
  TriggeredAt?: number | null;
}

File: Application.Domain/Model/Education/EducationSyncStatus.cs
namespace Application.Domain.Model.Education
{
    public class EducationSyncStatus
    {
        public string Id { get; set; } = "education/sync-status";
        public long? LastSyncAt { get; set; }
        public long? ExpiresAt { get; set; }
        public string? Status { get; set; }
        public string? Message { get; set; }
        public string? TriggeredByUserId { get; set; }
        public string? TriggeredByName { get; set; }
        public long? TriggeredAt { get; set; }
    }
}
